mysql-test/r/percona_log_slow_innodb.result

```diff
@@ -0,0 +1,50 @@
+DROP TABLE IF EXISTS t1, t2;
```

Comment:
Please remove this file, it has been moved to innodb suite, should not be re-added here

Reply:
@EvgeniyPatlan ping